OrbitClientModel/CaptureSerializerTest.cpp

```diff
@@ -70,7 +72,12 @@ TEST(CaptureSerializer, GenerateCaptureInfo) {
   selected_function.set_module_base_address(15);
   selected_functions[FunctionUtils::GetAbsoluteAddress(selected_function)] = selected_function;
 
+  TracepointInfo selected_tracepoint_info;
```

Comment:
Nit: I think it would improve readability if you would move this one line below, such that all operations on selected_tracepoint_info are together.
Feel free to ignore, though.

Reply:
I will update the code. Thank you!